docs/intro.rst

```diff
@@ -0,0 +1,25 @@
+Introdution
+===========
+
+MIS Builder is an Odoo module that implements a class of Odoo reports where KPI 
+(Key Performance Indicators)
+are displayed in rows, and time periods in columns. It focuses on very fast reporting
+on accounting data but can also use data from any other Odoo model.
+
+It features the following key characteristics:
+
+- User configurable: end users can create new report templates without development,
```

Comment:
The list is a little bit eclectic: it mixes functional features on one side and usability on the other. It would maybe be a good idea to split it in 2.